environment.yml

```diff
@@ -18,7 +18,8 @@ channels:
 dependencies:
 - python==3.6.8
 - pytorch==1.0.0
-- torchvision  
+- torchvision>=0.3.0
+- pycocotools>=2.0
```

Comment:
Not sure if pycocotools exists for windows. If not, I used this here before (but not sure this is the best way):
pip install "git+https://github.com/philferriere/cocoapi.git#egg=pycocotools&subdirectory=PythonAPI"

Reply:
https://github.com/cocodataset/cocoapi/issues/169#issuecomment-462528628
it might be the best way

Reply:
@jiata I got incompatible packages:
```console
$ conda env create -f environment.yml -n cvbp
Collecting package metadata (repodata.json): done
Solving environment: | 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed                                                                                                                                                                                                             
                                                                                                                                                                                                                   
UnsatisfiableError: The following specifications were found to be incompatible with each other:                                                                                                                    



Package pytorch conflicts for:
pytorch==1.0.0
fastai==1.0.48 -> pytorch[version='>=1.0.0']
torchvision[version='>=0.3.0'] -> pytorch[version='1.2.0|1.2.0+cu92|>=1.1.0']
fastai==1.0.48 -> torchvision -> pytorch[version='1.2.0|1.2.0+cu92|>=0.3|>=0.4|>=1.1.0']
```